lightning/src/routing/router.rs

```diff
@@ -778,17 +790,17 @@ pub fn get_route<L: Deref>(our_node_id: &PublicKey, network: &NetworkGraph, paye
 			if have_hop_src_in_graph {
 				// BOLT 11 doesn't allow inclusion of features for the last hop hints, which
 				// really sucks, cause we're gonna need that eventually.
-				let last_hop_htlc_minimum_msat: u64 = match hop.htlc_minimum_msat {
+				let last_path_htlc_minimum_msat: u64 = match hop.htlc_minimum_msat {
```

Comment:
I still don't understand this change.

Reply:
You mean the variable renaming or the checking along each hop path that the amount transferred is always above worst-hop's-htlc_minimum_msat-along-the-path ? Where such constraint only apply to hops after such value (i.e closer to payer).

Reply:
I don't understand the renaming.

Reply:
Honestly, I'm not sure either. I reverted the change.